### ISSUE TITLE
Dashed lines

### ISSUE THREAD

**Issue by martinthomson** [martinthomson/aasvg#10]
At the same time as #9, it probably makes sense to look into support for dashed lines drawn with U+2506 and similar.  There is already support for drawing them, just not recognizing them.

**Comment by rohan-wire** [comment]
First of all Martin, this tool is already so incredibly slick. I have not been this excited about something at IETF in a very long time. 
I am very interested in dashed and dotted lines. I'm guessing that there is a problem figuring out the semantics of period and colon characters, otherwise you could use those for horizontal and vertical dotted lines?

**Comment by martinthomson** [comment]
Yeah, the colons are easy.  Replicating the logic for recognizing vertical lines should work out reasonably well, with some tweaks (+ might not be something we want to allow at end points, I don't know).  Horizontal lines are tricky because period (.) is at the bottom of the box and there is a whole bit of logic that uses underscores for lines at the edge of the box.  We might need to use middle dot (·) or the box art options (┄ or ┅).

Mostly this just needs time, something I don't have much of, even though I enjoy this sort of low-pressure coding.
